### DISCUSSION TITLE
Can Renovate update inline bundler depencies

### DISCUSSION THREAD

**Discussion by mcritchlow** [renovatebot/renovate#13984] [Suggest an Idea]
For single file Ruby scripts, using this feature of Bundler is really useful.

My understanding is that updating Ruby dependencies currently in Renovate only looks at `Gemfile` and `Gemfile.lock`, so I'm just curious if something like this could be done.

Example:

```ruby
require 'bundler/inline'

gemfile do
  source 'https://rubygems.org'
  gem 'json', require: false
  gem 'nap', require: 'rest'
  gem 'cocoapods', '~> 0.34.1'
end

puts 'Gems installed and loaded!'
```

see: https://bundler.io/guides/bundler_in_a_single_file_ruby_script.html

**Comment by rarkins** [comment]
I think this could be supported. Do you name it differently from "Gemfile"? If so, have you tried adding it to `bundler.fileMatch` in case it works by luck? And if not, feel free to raise a feature request with a reproduction repo.

**Reply by mcritchlow** [reply]
Yeah, the name of the file could really be anything as I believe the primary use case of this is single-file scripts that need to pull in dependencies. I'll double check but I don't think it worked when I tried extending the `fileMatch`. I'll try again today and if I can confirm it doesn't work I'll open that feature request. Thanks!
